rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1716,15 +1719,15 @@ message ManageNeuron {
   // the permissions (i.e., principals that can change the neuron), the age, the followees, and
   // the dissolve state. The parent neuron's fees and maturity (if applicable in the given
   // nervous system) remain in the parent neuron and the child neuron's fees and maturity
-  // are initialized to be zero.
+  // are initialized to zero.
   message Split {
-    // The amount of governance tokens (in measured in fractions of 10E-8 of
+    // The amount of governance tokens (measured in fractions of 10e-8 of
     // a governance token) to be split to the child neuron.
     uint64 amount_e8s = 1;
 
     // The nonce that is used to compute the child neuron's
     // subaccount which also serves as the child neuron's ID. This nonce
-    // is also used as the memo field in the ledger transfer that transfers
+    // is also used as the memo field in the ledger transfer that transfers (Q: ledger transfer?)
```

Comment:
Could you clarify the question, please? 

"Ledger transfer" refers to the `icrc1_transfer` call that has an optional memo field in the request payload.